apps/web/vibes/soul/form/number-input/index.tsx

```diff
@@ -17,7 +17,16 @@ export const NumberInput = React.forwardRef<
   }
 >(
   (
-    { label, className, required, errors, decrementLabel, incrementLabel, disabled, ...rest },
+    {
+      label,
+      className,
+      required,
+      errors,
+      decrementLabel,
+      incrementLabel,
+      disabled = false,
```

Comment:
This should be the only change necessary, no? We don't also need to change the conditional rendering logic to "defend" against `undefined`

Reply:
Everything else was auto fixed by lint.

Reply:
LOL, pwned. Didn't know that was a thing 👍🏻

Reply:
😆